python/paddle/fluid/tests/unittests/test_parallel_executor_seresnext.py

```diff
@@ -367,6 +364,49 @@ def test_seresnext_with_fused_all_reduce(self):
         self._compare_with_fused_all_reduce(
             model=SE_ResNeXt50Small, use_cuda=False, iter=2, delta2=1e-3)
 
+    def test_seresnext_with_learning_rate_decay(self):
+        check_func_1 = partial(
+            self.check_network_convergence,
+            optimizer=optimizer,
+            use_parallel_executor=True)
+        check_func_2 = partial(
+            self.check_network_convergence,
+            optimizer=optimizer,
+            use_parallel_executor=False)
+        self._check_resnet_convergence(
+            SE_ResNeXt50Small,
+            check_func_1,
+            check_func_2,
+            use_cuda=True,
+            compare_seperately=False)
+        self._check_resnet_convergence(
+            SE_ResNeXt50Small,
+            check_func_1,
+            check_func_2,
+            use_cuda=False,
+            compare_seperately=False,
+            iter=2,
+            delta2=1e-3)
+
+    def test_seresnext_with_fused_optimizer_ops(self):
+        check_func_1 = partial(
+            self.check_network_convergence, fuse_all_optimizer_ops=False)
+        check_func_2 = partial(
+            self.check_network_convergence, fuse_all_optimizer_ops=True)
+        # self._check_resnet_convergence(
+        #     SE_ResNeXt50Small,
+        #     check_func_1,
+        #     check_func_2,
+        #     use_cuda=True,
+        #     delta2=1e-3)
+        self._check_resnet_convergence(
+            SE_ResNeXt50Small,
+            check_func_1,
+            check_func_2,
+            use_cuda=False,
+            iter=2,
+            delta2=1e-3)
```

Comment:
Add `fuse_all_optimizer_ops ` unit test in `test_parallel_executor_seresnext.py`.